lib/taskana-core/src/test/java/acceptance/task/UpdateTaskCommentAccTest.java

```diff
@@ -1,109 +1,149 @@
+import static pro.taskana.common.internal.util.CheckedSupplier.wrap;
 
-import acceptance.AbstractAccTest;
 import java.util.List;
 import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
+import testapi.TaskanaInject;
+import testapi.TaskanaIntegrationTest;
 
+import pro.taskana.classification.api.ClassificationService;
+import pro.taskana.classification.api.models.ClassificationSummary;
+import pro.taskana.common.api.TaskanaEngine;
 import pro.taskana.common.api.exceptions.ConcurrencyException;
 import pro.taskana.common.api.exceptions.NotAuthorizedException;
-import pro.taskana.common.test.security.JaasExtension;
+import pro.taskana.common.api.security.CurrentUserContext;
 import pro.taskana.common.test.security.WithAccessId;
 import pro.taskana.task.api.TaskService;
+import pro.taskana.task.api.models.ObjectReference;
+import pro.taskana.task.api.models.Task;
 import pro.taskana.task.api.models.TaskComment;
+import pro.taskana.task.internal.builder.TaskBuilder;
+import pro.taskana.task.internal.builder.TaskCommentBuilder;
 import pro.taskana.task.internal.models.TaskCommentImpl;
+import pro.taskana.workbasket.api.WorkbasketPermission;
+import pro.taskana.workbasket.api.WorkbasketService;
+import pro.taskana.workbasket.api.models.WorkbasketSummary;
+import pro.taskana.workbasket.internal.builder.WorkbasketAccessItemBuilder;
+
+@TaskanaIntegrationTest
+public class UpdateTaskCommentAccTest {
+
+  @TaskanaInject TaskService taskService;
+  @TaskanaInject TaskanaEngine taskanaEngine;
+  @TaskanaInject WorkbasketService workbasketService;
+  @TaskanaInject ClassificationService classificationService;
+  @TaskanaInject CurrentUserContext currentUserContext;
+
+  ClassificationSummary defaultClassificationSummary;
+  WorkbasketSummary defaultWorkbasketSummary;
+  ObjectReference defaultObjectReference;
+
+  @WithAccessId(user = "businessadmin")
+  @BeforeAll
+  void setup() throws Exception {
+    defaultClassificationSummary =
+        defaultTestClassification().buildAndStoreAsSummary(classificationService);
+    defaultWorkbasketSummary = defaultTestWorkbasket().buildAndStoreAsSummary(workbasketService);
+
+    // Set permissions for workbasket so it can be accessed by businessadmin
+    WorkbasketAccessItemBuilder.newWorkbasketAccessItem()
+        .workbasketId(defaultWorkbasketSummary.getId())
+        .accessId(currentUserContext.getUserid())
+        .permission(WorkbasketPermission.OPEN)
+        .permission(WorkbasketPermission.READ)
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+    defaultObjectReference = defaultTestObjectReference().build();
+  }
 
-@ExtendWith(JaasExtension.class)
-class UpdateTaskCommentAccTest extends AbstractAccTest {
-
-  @WithAccessId(user = "user-1-2")
+  @WithAccessId(user = "businessadmin")
```

Comment:
Is there a reason we run this test as businessadmin? Since businessadmin has further permissions I'd prefer using `user-1-1` and/or `user-1-2`